presto-hive/src/main/java/com/facebook/presto/hive/SyncPartitionMetadataProcedure.java

```diff
@@ -127,8 +141,17 @@ private void doSyncPartitionMetadata(ConnectorSession session, String schemaName
 
         try {
             FileSystem fileSystem = hdfsEnvironment.getFileSystem(context, tableLocation);
-            List<String> partitionsInMetastore = metastore.getPartitionNames(new MetastoreContext(session.getIdentity(), session.getQueryId(), session.getClientInfo(), session.getSource(), getMetastoreHeaders(session), isUserDefinedTypeEncodingEnabled(session), metastore.getColumnConverterProvider()), schemaName, tableName)
+            List<String> partitionNamesInMetastore = metastore.getPartitionNames(metastoreContext, schemaName, tableName)
```

Comment:
changing to `partitionNames` as a general term may be confusing because there are many other instance variables that hold account for partition names, I'd propose to keep current naming for this variable explicitly defined